.tekton/cli-main-ci-pull-request.yaml

```diff
@@ -51,28 +51,6 @@ spec:
         - name: kind
           value: task
         resolver: bundles
-    - name: show-summary
```

Comment:
This Task is not really useful and we may just drop it altogether. See more context in the comments of EC-643.